build.sbt

```diff
@@ -7,6 +7,18 @@ lazy val backend = (project in file("backend"))
   .dependsOn(core % "test->test;compile->compile")
   .settings(backendSettings:_*)
 
+lazy val localBackend = (project in file("local-backend"))
```

Comment:
Sweet. 

I think the SBT magic I was referring to could be handled by setting up whichever dir you set up as its own SBT project, but there might need to be some extra glue at the root to make it work right. Let me know if it's problematic and I can take a closer look.